frontend/src/app/components/filter-section/filter-section.component.ts

```diff
@@ -0,0 +1,112 @@
+import { Component, OnChanges, OnInit, SimpleChanges } from '@angular/core';
+
+@Component({
+  selector: 'app-filter-section',
+  templateUrl: './filter-section.component.html',
+  styleUrls: ['./filter-section.component.sass']
+})
+export class FilterSectionComponent implements OnInit {
+
+  public showLangDropdown = false;
+  public showTopicsDropdown = false;
+
+  public selectedLanguageCount = 0;
+  public selectedTopicsCount = 0;
+
+  public selectedLanguageFilters = new Set();
+  public selectedTopicsFilters = new Set();
```

Comment:
We need to create a general dropdown component and reuse it in all places where we need it